zhmcclient/_session.py

```diff
@@ -1785,7 +1784,13 @@ def wait_for_completion(self, operation_timeout=None):
             start_time = time.time()
 
         while True:
-            job_status, op_result_obj = self.check_for_completion()
+            try:
+                job_status, op_result_obj = self.check_for_completion()
+            except ConnectionError:
+                HMC_LOGGER.debug("Got ConnectionError while waiting for "
+                                 "completion of job %s. This could be "
+                                 "because HMC is restarting.", self.uri)
```

Comment:
I suggest that the log message makes it clear that a retry is performed. Something like: "Retrying after ConnectionError while waiting for ...."

Reply:
Done